packages/web/src/app/admin/orders/utils/formatters.ts

```diff
@@ -0,0 +1,9 @@
+export const formatPrice = (priceInCents: number): string =>
+    (priceInCents / 100).toLocaleString("de-DE", { style: "currency", currency: "EUR" });
```

Comment:
Duplicate function. This already exists in `Utils.price.display(...)` in `/packages/web/src/lib/utils/price.tsx`